### PR TITLE
➕ Add TypeScript

### PR DESCRIPTION
**Pull request by cboenke**
- Created `tsconfig.json` with: `touch tsconfig.json`
- Ran `next` with: `npm run dev`
- Installed required packages with: `npm install --save-dev typescript @types/react @types/node`
- Renamed `_app.js` and `index.js` to `_app.tsx`and `index.tsx`
- Renamed `hello.js`to `hello.ts`
- Configured `tsconfig.json`with: `npm run dev`
- Restarted server with `npm run dev`

https://nextjs.org/docs/basic-features/typescript

### CODE REVIEW

**Review by m-wiemers — APPROVED** [review]
A ray of sunshine for the eyes ☀😎

**Comment by DNaunin** [comment]
LGTM! 🚀  Great description.

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "jsx": "preserve"
+  },
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx"
+  ],
+  "exclude": [
+    "node_modules"
+  ]
```
Comment (phlgr) [inline tsconfig.json]: These lines don't seem to be prettified.

**Review by phlgr — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 1 above)
Comment (cboenke) [reply tsconfig.json]: Thanks! Are prettified now.

**Review by mrana17 — APPROVED** [review]
lgtm 🚀

**Review by phlgr — APPROVED** [review]
LGTM 🚀

**Pull request merged**